src/Microsoft.Health.Fhir.Api/Features/Audit/AuditEventTypeMapping.cs

```diff
@@ -0,0 +1,67 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using EnsureThat;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc.Controllers;
+using Microsoft.AspNetCore.Mvc.Infrastructure;
+using Microsoft.Health.Extensions.DependencyInjection;
+using Microsoft.Health.Fhir.Api.Extensions;
+
+namespace Microsoft.Health.Fhir.Api.Features.Audit
+{
+    /// <summary>
+    /// Provides the ability to lookup audit event type.
```

Comment:
Might want to provide a little more context here and why this solution is necessary

Reply:
That's a good idea. I will add the comments.